sycl/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ include(AddSYCLExecutable)
 set(SYCL_MAJOR_VERSION 2)
 set(SYCL_MINOR_VERSION 1)
 set(SYCL_PATCH_VERSION 0)
-set(SYCL_DEV_ABI_VERSION 5)
+set(SYCL_DEV_ABI_VERSION 4)
```

Comment:
Gail, looks like you reverted the version change by mistake.

Reply:
It was the ABI version that I updated, and that I am now reverting.